app/src/main/java/org/mozilla/fenix/compose/Divider.kt

```diff
@@ -0,0 +1,154 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.compose
+
+import android.content.res.Configuration
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxHeight
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import org.mozilla.fenix.theme.FirefoxTheme
+
+/**
+ * Generic Divider
+ *
+ * @param modifier [Modifier] Used to shape and position the underlying [androidx.compose.material.Divider]
+ * @param color  [Color] of the divider line
+ */
+@Composable
+fun Divider(
+    modifier: Modifier = Modifier,
+    color: Color = FirefoxTheme.colors.borderPrimary,
+) {
+    androidx.compose.material.Divider(
+        modifier = modifier,
+        color = color,
+        thickness = 0.5.dp,
+    )
+}
+
+/**
+ *
+ */
+@Composable
+@Preview(uiMode = Configuration.UI_MODE_NIGHT_YES)
+@Preview(uiMode = Configuration.UI_MODE_NIGHT_NO)
+private fun HorizontalDividerPreview() {
+    FirefoxTheme {
+        Box(Modifier.background(FirefoxTheme.colors.layer1)) {
+            Column {
+                Text(text = "Above the line")
+
+                Divider(modifier = Modifier.padding(vertical = 10.dp, horizontal = 24.dp))
+
+                Text(text = "Below the line")
+            }
+        }
+    }
+}
+
+/**
+ * To use the vertical divider, add Modifier.fillMaxHeight().width(0.5.dp) and
+ * add any additional modifiers after.
```

Comment:
I would generally try to avoid repeating the code in the kDoc. Also, the 0.5dp will also be a bit out of date given the comment above. At the very least, I wouldn't try to be prescriptive in the kDoc for a @Preview since this would mean we would always have to change this code in the comment to reflect any changes to the Divider component.

Reply:
ah that makes sense, I'll remove the code from the kDoc